packages/common/core-utils/src/test/promiseCache.spec.ts

```diff
@@ -5,7 +5,7 @@
 
 import { strict as assert } from "node:assert";
 import { SinonFakeTimers, useFakeTimers } from "sinon";
-import { PromiseCache } from "../..";
+import { PromiseCache } from "../promiseCache";
 
```

Comment:
Tests that weren't reaching into internal modules should not start now.
Update the config if needed to allow self reference.